packages/terra-dropdown-button/tests/wdio/split-button-spec.js

```diff
@@ -159,19 +159,19 @@ Terra.describeViewports('Split Button', ['medium'], () => {
         Terra.validates.element('click');
       });
 
-      it('keyboard navigates down past the last option', () => {
-        browser.keys(['ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'Space']);
-        Terra.validates.element('no down out of bounds');
+      it('Navigates from last item to first item', () => {
+        browser.keys(['ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'Enter']);
+        Terra.validates.element('first item focused');
```

Comment:
Same here about the focus test:

Reply:
Added test case to focus first list item.